src/unicef_vision/synchronizers.py

```diff
@@ -37,16 +38,14 @@ def _convert_records(self, records):    # pragma: no cover
     def _save_records(self, records):   # pragma: no cover
         pass
 
-    @abstractmethod
-    def _get_kwargs(self):  # pragma: no cover
-        return {}
+    def set_kwargs(self, *args, **kwargs):
```

Comment:
`*args` are not being used, any reason to be passing them in to this method?

Reply:
we can remove them